resources/js/Pages/Admin/Roles/CreateRoleForm.vue

```diff
@@ -63,6 +63,7 @@
 <script>
     import JetActionMessage from '@src/Jetstream/ActionMessage';
     import JetFormSection from '@src/Jetstream/FormSection';
+    import JetCheckbox from '@src/Jetstream/Checkbox';
```

Comment:
```suggestion
    import XCheckbox from '@src/Components/Checkbox';
```

Implement this change as mentioned above (including the component move)

Reply:
still a jetstream component